jena-text/src/main/java/org/apache/jena/query/text/TextDatasetFactory.java

```diff
@@ -27,7 +27,7 @@
 import org.apache.jena.system.JenaSystem ;
 import org.apache.lucene.analysis.Analyzer;
 import org.apache.lucene.store.Directory ;
-import org.apache.solr.client.solrj.SolrServer ;
+import org.elasticsearch.indices.IndexCreationException;
```

Comment:
You should not need this import here. Exceptions in index creation should be thrown as TextIndexExceptions from within TextIndexES.